models/green_oracle/src/training/training.py

```diff
@@ -0,0 +1,69 @@
+import pandas as pd
+from pathlib import Path
+
+from views_runs import Storage, StepshiftedModels
+from views_partitioning.data_partitioner import DataPartitioner
+from viewser import Queryset, Column
+from views_runs import operations
+from views_runs.run_result import RunResult
+from views_runs import storage
+from views_runs.storage import store, retrieve, fetch_metadata
+from viewser.operations import fetch
+import views_runs
+from views_partitioning import data_partitioner, legacy
+from stepshift import views
+
+
+
+from ...configs import config
+from ..dataloaders.queryset import get_querysets
+
+def training():
+    model = config.model
+    modelstore = storage.Storage()
+    print('Calibration partition')
+    model['Algorithm_text'] = str(config.model['algorithm'])
+    
+    model['RunResult_calib'] = RunResult.retrain_or_retrieve(
+        retrain=bool(config.common_config['force_retrain']),
```

Comment:
We'll go another way - please consult Xiaolong or his code in the stepshifter branch (if deleted, it is merged with main)

Reply:
Hi, thanks! baseline models will not require this for now and so I'll take the latest trainer for future models.